test/services/DataGeneratorSpec.scala

```diff
@@ -232,65 +272,36 @@ class DataGeneratorSpec extends PlaySpec with CSVTestData with GuiceOneAppPerSui
         schemeType = "CSOP"
       )
       val result = intercept[ERSFileProcessingException] {
-        dataGenerator.getData(XMLTestData.getCSOPWithoutData)(schemeInfo, hc, request)
+        dataGenerator.getErrors(XMLTestData.getCSOPWithoutData)(schemeInfo, hc, request)
       }
       result.message mustBe "The file that you chose doesn’t have any data after row 9. The reportable events data must start in cell A10.<br/><a href=\"https://www.gov.uk/government/collections/employment-related-securities\">Use the ERS guidance documents</a> to help you create error-free files."
     }
 
     "get Data for Iterator of Strings" in {
-      val result = dataGenerator.getData(XMLTestData.getEMIAdjustmentsTemplate)(schemeInfo,hc,request)
+      val result = dataGenerator.getErrors(XMLTestData.getEMIAdjustmentsTemplate)(schemeInfo,hc,request)
       result.size must be (1)
       result.foreach(_.data.foreach(_ mustBe (XMLTestData.emiAdjustmentsExpData)))
       try {
-        dataGenerator.getData(XMLTestData.getIncorrectsheetNameTemplate)(schemeInfo,hc,request)
+        dataGenerator.getErrors(XMLTestData.getIncorrectsheetNameTemplate)(schemeInfo,hc,request)
       } catch {
         case e:Throwable => e.getMessage mustBe "Incorrect ERS Template - Sheet Name isn't as expected"
       }
       verify(mockAuditEvents, times(1)).fileProcessingErrorAudit(argEq(schemeInfo), argEq("EMI40_Adjustment"), argEq("Could not set the validator"))(any(), any())
     }
 
     "get mandatory Data for Iterator of Strings" in {
-      val result = dataGenerator.getData(XMLTestData.getEMIReplacedTemplate)(schemeInfo,hc,request)
+      val result = dataGenerator.getErrors(XMLTestData.getEMIReplacedTemplate)(schemeInfo,hc,request)
       result.size must be (1)
       result.foreach(_.data.foreach(_ must be (XMLTestData.emiReplacedExpMandatoryData)))
     }
 
     "expand repeated rows" in {
-      val result = dataGenerator.getData(XMLTestData.getEMIAdjustmentsRepeatedTemplate)(schemeInfo,hc,request)
+      val result = dataGenerator.getErrors(XMLTestData.getEMIAdjustmentsRepeatedTemplate)(schemeInfo,hc,request)
       result.size mustEqual(1)
       result(0).data.size mustEqual(4)
```

Comment:
Scalastyle: **result(0)**.data.size mustEqual(4)

```suggestion
      result.head.data.size mustEqual(4)
```